.github/workflows/create_wappalyzer_update.yml

```diff
@@ -0,0 +1,45 @@
+name: Create Wappalyzer Update PR
+
+on:
+  push:
+    paths:
+      README.md
+  schedule:
+    - cron:  '0 4 3 * *'
+
+jobs:
+  create_pr:
+    name: Create Pull Request
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Feature Branch and Raise PR
+      run: |
+        # Setup git details
+        export GITHUB_USER=zapbot
+        git config --global user.email "zapbot@zaproxy.org"
+        git config --global user.name $GITHUB_USER
+        # Clone repos
+        git clone https://github.com/AliasIO/wappalyzer.git
```

Comment:
`--depth 1` ?

Reply:
For zap-extensions as well?

Reply:
Was just suggesting for this repo, the one that will be pushed/PRed needs the whole history (at least the last time I tried push/PR GitHub didn't like the partial history).